From 859e09f466c94329c538456ba9b125ca4dc672f5 Mon Sep 17 00:00:00 2001
From: Jonghwa Lee <jonghwa3.lee@samsung.com>
Date: Fri, 1 Nov 2013 14:54:23 +0900
Subject: [PATCH 0917/1302] Revert "Thermal: exynos: Support for TMU regulator
 defined at device tree"

This reverts commit ed58c6a559cf9a870e55587872f9b0eb322e3e6b.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/thermal/exynos_thermal.c | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/drivers/thermal/exynos_thermal.c b/drivers/thermal/exynos_thermal.c
index 84794b3..788b1dd 100644
--- a/drivers/thermal/exynos_thermal.c
+++ b/drivers/thermal/exynos_thermal.c
@@ -38,7 +38,6 @@
 #include <linux/cpufreq.h>
 #include <linux/cpu_cooling.h>
 #include <linux/of.h>
-#include <linux/regulator/consumer.h>
 
 /* Exynos generic registers */
 #define EXYNOS_TMU_REG_TRIMINFO		0x0
@@ -118,8 +117,6 @@
 
 #define EXYNOS_ZONE_COUNT	3
 
-#define EXYNOS_TMU_REGULATOR "vdd_ts"
-
 struct exynos_tmu_data {
 	struct exynos_tmu_platform_data *pdata;
 	struct resource *mem;
@@ -903,7 +900,6 @@ static int exynos_tmu_probe(struct platform_device *pdev)
 {
 	struct exynos_tmu_data *data;
 	struct exynos_tmu_platform_data *pdata = pdev->dev.platform_data;
-	struct regulator *reg;
 	int ret, i;
 
 	if (!pdata)
@@ -913,21 +909,6 @@ static int exynos_tmu_probe(struct platform_device *pdev)
 		dev_err(&pdev->dev, "No platform init data supplied.\n");
 		return -ENODEV;
 	}
-
-	reg = regulator_get(&pdev->dev, EXYNOS_TMU_REGULATOR);
-	if (!IS_ERR(reg)) {
-		ret = regulator_enable(reg);
-		if (ret) {
-			dev_err(&pdev->dev, "Regulator %s not enabled.\n",
-				EXYNOS_TMU_REGULATOR);
-			return ret;
-		}
-	} else {
-		dev_warn(&pdev->dev,
-			 "Regulator %s not defined at device tree.\n",
-			 EXYNOS_TMU_REGULATOR);
-	}
-
 	data = devm_kzalloc(&pdev->dev, sizeof(struct exynos_tmu_data),
 					GFP_KERNEL);
 	if (!data) {
-- 
1.8.3.2

